compiler/src/dmd/backend/x86/cgcod.d

```diff
@@ -1818,8 +1818,8 @@ regm_t lpadregs()
 void useregs(regm_t regm)
 {
     //printf("useregs(x%llx) %s\n", regm, regm_str(regm));
-    assert(REGMAX <= 32);
-    regm &= (1UL << REGMAX) - 1;
+    if (REGMAX < 64)
+        regm &= (1UL << REGMAX) - 1;
```

Comment:
this looks like an out of bounds shift. Also this branch is dead?

Reply:
I changed it to a static if.